### DISCUSSION TITLE
Something like FaunaDB or DataStax Astra DB possible?

### DISCUSSION THREAD

**Discussion by ivan-velikanov** [ArcadeData/arcadedb#48] [Ideas]
My use case is such, I want to run web services completely free. I need to have a mobile/desktop app (JavaFX) and they need to work together on some data classification work (some work related to a non-paying NGO). This project will run for decades, I cannot give them a paid solution it will not work, the bills may not be cleared and the system will die. The only alternative they have is Google spreadsheets and the API to manage somehow (for free). Other alternatives are Fauna DB (too strict quotas) and DataStax Astra DB. 
While OrientDB / Astra DB is nice. There is no scope that this project can use this other than for local storage in the mobile/desktop app, the cloud component of this service will have to rely on one of the following - Google Sheets, Heroku free Postgres, FaunaDB, or DataStax Astra DB. 
Long story short, the decision here is not based on technology, but based on what is free. 
This allows people who work on a zero-budget basis to launch themselves. Now think about it, if I launched my product based on, say, Cassandra and DataStax Astra DB in the cloud, now to have a local offline database in something like orientdb/arcadedb means knowing two technologies. I mean, such a niche might not be a target for ArcadeDB right now. But just wanted to highlight the point that technology/engineering is not the only reason for choosing/selecting a solution. Same way quite often, for saving on hosting, we would be compelled to use PHP, because that is easy and cheap to get compare to java. Only recently with Heroku, we started seeing Java/JVM-based cloud hosting for free.

**Comment by lvca** [comment]
Hi @ivan-velikanova if I'm not wrong, the Heroku free tier allows you to tun a docker image. So you could spin an ArcadeDb server in a few seconds. So this is already possible right now. Of course, it's not going to be a managed instance with live support, etc. (such costs need to be covered somehow).

You could try to run the command below and then open the browser on that server, port 2480, to see if you can see ArcadeDB studio:

```
docker run --rm -p 2480:2480 -p 2424:2424 --env arcadedb.server.rootPassword=playwithdata --env "arcadedb.server.defaultDatabases=Imported[root]{import:https://github.com/ArcadeData/arcadedb-datasets/raw/main/orientdb/OpenBeer.gz}" arcadedata/arcadedb:latest
```

**Reply by ivan-velikanov** [reply]
But where would the data be stored? The hard disk storage of Heroku is not persistent.

**Reply by lvca** [reply]
My idea is to export/import transparently the database on public FREE places like GitHub. I contacted you on Twitter if you want to chat about it.
